### ISSUE TITLE
Recent change to devel will prevent auto merge to Master.

### ISSUE THREAD

**Issue by jpvandy** [sstsimulator/sst-elements#651]
A change in the last 36 hours (April 27 or later) is issuing a non-fatal error message from a number of test Suites that will prevent auto merge to Master until it is fixed (or bailing wire is added to the testing structure.)     Two of the tests displaying it are EmberSweep and merlin 3 of the 5 merlin tests.
The message from sst is: 
```
Opening element library sst failed: file not found Opening and resolving references for element library sst failed:
        /home/jwilso/jenkins/workspace/SST__Nightly_Multi_Rank_sst-test_full_n-2/487_487/local/sst-elements/lib/sst-elements-library/libsst.so: cannot open shared object file: No such file or directory
```

**Comment by jpvandy** [comment]
@nmhamster 
@allevin

**Comment by jpvandy** [comment]
This was fixed before the 7.0 release
